terra-env/apps.tf

```diff
@@ -74,3 +74,23 @@ module "workspace_manager" {
     google-beta.target = google-beta.target
   }
 }
+
+module "crl_janitor" {
+  source = "github.com/broadinstitute/terraform-ap-modules.git//crl-janitor?ref=yyu-CA-870"
```

Comment:
you are going to want this to reference a release tag but otherwise this lgtm now.